src/Routing/Router.js

```diff
@@ -25,7 +25,7 @@ const Router = ({history, isLoggedIn}) => {
                     <UnRestrictedRoute exact path="/reset-password" component={ResetPassword} isLoggedIn={isLoggedIn}/>
 
                     <RestrictedRoute exact path="/dashboard" component={Dashboard} isLoggedIn={isLoggedIn}/>
-                    <RestrictedRoute exact path="/profile" component={Profile} isLoggedIn={isLoggedIn}/>
+                    <RestrictedRoute exact path="/profile" component={Profile} isLoggedIn={true}/>
```

Comment:
here Why it's static value isLoggedIn={true} it should be  isLoggedIn={isLoggedIn}?